code/controllers/subsystem/weeds.dm

```diff
@@ -8,6 +8,8 @@ SUBSYSTEM_DEF(weeds)
 	var/list/creating = list()
 	var/list/pending = list()
 	var/list/currentrun
+	/// The amount of time a weed will try to spawn on a given tile
+	var/list/attempt = list()
```

Comment:
I discovered that some weeds could indefinitly try to spawn, until their node is destroyed